### DISCUSSION TITLE
wordpress

### DISCUSSION THREAD

**Discussion by giornalemotori** [f1db/f1db#5] [Q&A] [answered]
hi, how can i use on a wordpress site?
thanks

**Comment by marceloverdijk** [comment, marked as answer]
Hi @giornalemotori 

Thanks for reaching out. The F1DB data can be downloaded as json or as an sqlite database.
So in Wordpress you should either parse the json data or query from the sqlite database.
I'm not familiar with Wordpress so I don't know if either is possible.

Another approach is to parse the json outside Wordpress (locally?) and iterate over the data and do what ever is needed to build your Wordpress website. E.g you could create pages or push the content somewhere from which your Wordpress instance can retrieve it.

Does that help?
